packages/nodes-base/nodes/Disqus/Disqus.node.ts

```diff
@@ -65,22 +65,22 @@ export class Disqus implements INodeType {
 					{
 						name: 'Get',
 						value: 'get',
-						description: 'Returns forum details.',
+						description: 'Retrieve forum details.',
```

Comment:
I think that the change from Return to Retrieve here is unnecessary. Can you please change that and remove the full stops?